lib/commands/tunnel.ts

```diff
@@ -39,17 +39,25 @@ export default class TunnelCmd extends Command {
 	public static description = stripIndent`
 		Tunnel local ports to your balenaOS device.
 
-		Use this command to open local ports which tunnel to listening ports on your balenaOS device.
+		Use this command to open local TCP ports that tunnel to listening sockets in a
+		balenaOS device.
```

Comment:
Some of the changes were for the lines to fit in 80 characters. Some changes were minor, for improved clarity (hopefully).